lib/cisco_node_utils/cmd_ref/vni.yaml

```diff
@@ -0,0 +1,51 @@
+# vni
+---
+all_vnis:
+  config_get: "show vni"
+  config_get_token: '/^(\d+)\s/'
+
+bridge_domain:
+  config_get: "show vni"
```

Comment:
Most of the time it's better to get state from running-config instead of exec commands. Is there a reason why you can't get it from running-config?